sale_order_line_date/models/sale_order_line.py

```diff
@@ -0,0 +1,28 @@
+# © 2016 OdooMRP team
+# © 2016 AvanzOSC
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# © 2016 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    commitment_date = fields.Datetime(
+        related="order_id.commitment_date",
```

Comment:
Thinking about it, do we really want to force the commitment date on each line to the one defined on sale.order.
I feel like setting different commitment dates between the lines is a real feature which is probably used in former version, so defining this as related here looks like a regression :thinking:

Reply:
Exactly, you are changing the functionality of the module doing this. Respect the write and create methods that were there before as well.

Reply:
Agree here with @lreficent